user-guide/Troubleshooting/Known_issues/KI_Multiple_Datacenters.md

```diff
@@ -0,0 +1,39 @@
+---
+uid: KI_Multiple_Datacenters
+---
+
+# Various issues with a geo-redundant Cassandra setup with multiple data centers
+
+## Affected versions
+
```

Comment:
For customers, affected versions could be simply:
-geo redundant or site redundant Cassandra setups using multiple datacenters.
We can skip the more technical explanations.